### ISSUE TITLE
Rails 5.2

### ISSUE THREAD

**Issue by yapus** [amatsuda/erd#20]
```
class << self
       def status
         migrated_versions = ActiveRecord::Base.connection.select_values("SELECT version FROM #{ActiveRecord::Migrator.schema_migrations_table_name}").map {\|v\| '%.3d' % v}
         migrations = []
         ActiveRecord::Migrator.migrations_paths.each do \|path\|
           Dir.foreach(Rails.root.join(path)) do \|file\|
--
```
vendor/bundle/ruby/2.5.0/gems/erd-0.4.0/lib/erd/migrator.rb:9:in `status'
vendor/bundle/ruby/2.5.0/gems/erd-0.4.0/app/controllers/erd/erd_controller.rb:19:in `index'

**Comment by brianmcmichael** [comment]
+1

**Comment by amatsuda** [comment]
Resolved via #22
